src/org/anjocaido/groupmanager/permissions/AnjoPermissionsHandler.java

```diff
@@ -702,8 +687,6 @@ public int getPermissionInteger(String user, String variable) {
 				for (Group subGroup : auser.subGroupListCopy()) {
 					result = nextGroupWithVariable(subGroup, variable);
 					// Found value?
-					if (result != null)
-						continue;

```

Comment:
This looks like a bug and should be a Break not a Continue. Needs Testing.

Reply:
Looks like fixing this bug is out of scope of this pull request. For now I'll just revert it back, we'll have to look into it later